routers/matchmaking.js

```diff
@@ -208,7 +208,7 @@ async function GetInstances(RoomId = null) {
     if(RoomId === null) {
         var instances = [];
         Object.keys(SubroomInstances).forEach(element => {
-            instances.concat(SubroomInstances[element]);
+            instances = instances.concat(SubroomInstances[element]);
```

Comment:
This sets the value of `instances`, as I was not aware at the time that Array methods like these are actually immutable.